test-utils/src/index.js

```diff
@@ -28,6 +28,21 @@ export function act(cb) {
 	if (flush) {
 		flush();
 	}
-	options.debounceRendering = Component.__test__previousDebounce;
 	options.requestAnimationFrame = previousRequestAnimationFrame;
 }
+
+/**
+ * Teardown test environment and reset preact's internal state
+ */
+export function teardown() {
+	if (Component.__test__drainQueue) {
+		// Flush any pending updates leftover by test
+		Component.__test__drainQueue();
+		delete Component.__test__drainQueue;
+	}
+
+	if (typeof Component.__test__previousDebounce !== 'undefined') {
+		options.debounceRendering = Component.__test__previousDebounce;
```

Comment:
Same deal for `Component.__test__previousDebounce` - seems like these belong on `options`, the Component thing was inherited from when `options.debounceRendering` got renamed to `Component.debounce` in ceviche.